pubspec.yaml

```diff
@@ -43,7 +43,6 @@ dependencies:
   image_cropper: ^1.4.1
   carousel_slider: ^4.0.0
   dots_indicator: ^2.0.0
-  permission_handler: ^8.0.1
```

Comment:
I removed `permission_handler` package because apparently it is not being used (I did a search on the app). 

Apparently it was introduced to obtain camera permissions when you start the scanner in the legacy bloc scanner, but it is not being used anymore.

Reply:
Well Not sure if we want to re-introduce this permission service.
from the PR https://github.com/JoinSEEDS/seeds_light_wallet/pull/236

> QR Scanner infrastructure.
> Using QR Scanner for scanning either invite codes or invite URLs.
> If the user denies access to the camera the Scan QR button will be replaced with info about denied permissions including a link to open the System Settings if the user changes their mind.